src/e2e.js

```diff
@@ -135,12 +135,14 @@ mye2e.prototype.signEncrypt = function(data, encryptKey, sign) {
     if (typeof sign === 'undefined') {
       sign = true;
     }
-    var importResult = e2e.async.Result.getValue(
-      pgp.importKey(function(str, f) {
-        f('');
-      }, encryptKey));
-    var keys = e2e.async.Result.getValue(
-      pgp.searchPublicKey(importResult[0]));
+    var keys = [];
+    if (encryptKey) {
+      var importResult = e2e.async.Result.getValue(
+        pgp.importKey(function(str, f) {
+          throw new Error('No passphrase for a public key');
```

Comment:
I know this is nitty, but may be good to say "No passphrase needed for a public key".